index.js

```diff
@@ -33,7 +33,12 @@ exports.getFile = function (filePath, preserveFormatting, cb) {
   cb = once(cb)
   fs.createReadStream(filePath, { encoding: 'utf8' })
     .pipe(split())
-    .pipe(through(online))
+    .pipe(new Writable({
+      write: (chunk, encoding, callback) => {
```

Comment:
as would arrow functions
```suggestion
      write: function (chunk, encoding, callback) {
```

Reply:
see above, realistically hostile already requires node 10+ due to chalk usage